code/game/objects/structures/tables_racks.dm

```diff
@@ -870,3 +870,17 @@
 		R.add_fingerprint(user)
 		qdel(src)
 	building = FALSE
+
+/obj/item/rack_parts/attack_self_tk(mob/user)
```

Comment:
I feel like it would be possible to instead have rack parts build where it is (so src) instead of user.loc, because this is just copy paste?

Reply:
okie